src/utils/providers.ts

```diff
@@ -3,6 +3,6 @@ import getRpcUrl from 'utils/getRpcUrl'
 
 const RPC_URL = getRpcUrl()
 
-export const simpleRpcProvider = new ethers.providers.StaticJsonRpcProvider(RPC_URL)
+export const simpleRpcProvider = new ethers.providers.JsonRpcBatchProvider(RPC_URL)
```

Comment:
If we use that it sends too much requests to get chain id from node which is known already and always the same.

Reply:
https://github.com/pancakeswap/pancake-frontend/pull/2571

Reply:
yea, I was trying this batch provider, everything seems fine apart from this.